src/utl/include/utl/Logger.h

```diff
@@ -59,6 +59,9 @@
 
 namespace utl {
 
+class PrometheusMetricsServer;
+class Registry;
```

Comment:
Registry is a very generic name.  I would prefer to see the code the prometheus subdir in namespace like utl::prometheus

Reply:
I think that's too much name spacing for this code, especially if it's going to be called in a lot of places. I renamed it to PrometheusRegistry, everything else feels uniquely named enough to live in utl.